public/app/plugins/datasource/loki/querybuilder/components/LokiQueryBuilder.test.tsx

```diff
@@ -51,6 +51,23 @@ describe('LokiQueryBuilder', () => {
     await waitFor(() => expect(screen.getByText('job')).toBeInTheDocument());
   });
 
+  it('does not show already existing label names as option in label filter', async () => {
+    const props = createDefaultProps();
+    props.datasource.getDataSamples = jest.fn().mockResolvedValue([]);
+    props.datasource.languageProvider.fetchSeriesLabels = jest
+      .fn()
+      .mockReturnValue({ job: ['a'], instance: ['b'], baz: ['bar'] });
+
+    render(<LokiQueryBuilder {...props} query={defaultQuery} />);
+    await userEvent.click(screen.getByLabelText('Add'));
+    const labels = screen.getByText(/Label filters/);
+    const selects = getAllByRole(labels.parentElement!.parentElement!.parentElement!, 'combobox');
```

Comment:
First time that I see this in a RTT test. Is this for interacting with the `select`? Maybe try `import { selectOptionInTest } from 'test/helpers/selectOptionInTest';` for that?

Reply:
Great suggestion! Updated in https://github.com/grafana/grafana/pull/57187/commits/52f334fafe96577046e246e3ee465c8a2433d3b5